### DISCUSSION TITLE
Do We Have Modals & Hybrid Commands?

### DISCUSSION THREAD

**Discussion by HarshPatel5940** [Pycord-Development/pycord#1362] [Q&A] [answered]
If yes, can someone just hook me up with an example of a modal?

**Comment by lorddusk** [comment, marked as answer]
https://github.com/Pycord-Development/pycord/blob/master/examples/modal_dialogs.py

**Reply by HarshPatel5940** [reply]
Thanks 👍

**Comment by plun1331** [comment]
You can find examples in the examples directory. [Here's the example for modals](https://github.com/Pycord-Development/pycord/blob/master/examples/modal_dialogs.py).

**Comment by HarshPatel5940** [comment]
Thanks 👍
